cram/cram_codecs.c

```diff
@@ -558,7 +558,7 @@ cram_codec *cram_beta_decode_init(char *data, int size,
     else if (option == E_BYTE_ARRAY || option == E_BYTE)
 	c->decode = cram_beta_decode_char;
     else {
-	fprintf(stderr, "BYTE_ARRAYs not supported by this codec\n");
+	hts_log_error("BYTE_ARRAYs not supported by this codec");
```

Comment:
I'd rather not change the formatting as part of a pull request targeting something else, so I've tried to stick to whatever was in the file already.

But I totally agree that the mix of tabs and spaces is horrible. I will gladly make a pull request covering ONLY tabs-to-spaces reformatting if people agree it's a good idea.